src/devices/cpu/hcd62121/hcd62121.h

```diff
@@ -98,9 +102,14 @@ class hcd62121_cpu_device :  public cpu_device
 	u8 m_f;
 	u8 m_time;
 	u8 m_time_op;
+	u8 m_unk_f5;
 	s32 m_cycles_until_timeout;
 	bool m_is_timer_started;
+	bool m_is_timer_irq_enabled;
+	bool m_is_timer_irq_asserted;
+	bool m_is_timer_wait_elapsed;
 	bool m_is_infinite_timeout;
+	s32 m_timer_cycles;
 	emu_timer *m_timer;
```

Comment:
You seem to have neglected to register these for save states in `device_start`.

Reply:
Thanks, addressed here: https://github.com/mamedev/mame/pull/13144